production/readOnly-openAPI.json

```diff
@@ -30508,10 +30508,10 @@
           }
         ]
       },
-      "ProductsListResponse": {
+      "ProductsListResponseBody": {
```

Comment:
To nie powinno być w OpenAPI.json zamiast readOnly?

Reply:
na read only nie patrzymy w PR-kach :)

Reply:
No ale nie powinno to być poprawione w oryginalnym pliku. Bo tam jest np. cały czas ProductsListResponse